packages/neuron-ui/src/styles/layout.scss

```diff
@@ -1,5 +1,5 @@
 :root{
-  --sidebar-width: 150px;
+  --sidebar-width: 158px;
```

Comment:
What's the sidebar width from the design? A bit wider?

Reply:
> What's the sidebar width from the design? A bit wider?

It's from the last design in zeplin.